ndbench-es-plugins/src/main/java/com/netflix/ndbench/plugin/es/EsPlugin.java

```diff
@@ -0,0 +1,134 @@
+/*
+ *  Copyright 2016 Netflix, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package com.netflix.ndbench.plugin.es;
+
+import com.google.inject.Singleton;
+import com.netflix.ndbench.api.plugin.DataGenerator;
+import com.netflix.ndbench.api.plugin.NdBenchClient;
+import com.netflix.ndbench.api.plugin.annotations.NdBenchClientPlugin;
+import org.elasticsearch.action.get.GetResponse;
+import org.elasticsearch.client.transport.TransportClient;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.common.transport.InetSocketTransportAddress;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetAddress;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * @author vchella
+ */
+@Singleton
+@NdBenchClientPlugin("ElasticSearchPlugin")
+public class EsPlugin implements NdBenchClient {
+
+    private static final Logger logger = LoggerFactory.getLogger(EsPlugin.class);
+
+    private DataGenerator dataGenerator;
+
+    private TransportClient client;
+
+    private static final String ResultOK = "Ok";
+    private static final String CacheMiss = null;
+    private String EsIndexName ="blog_index_";
+
+    private String ClusterName="Localhost",
```

Comment:
Shouldn't the `ClusterName` be defined as `static` then? Shouldn't we give a more representative name like `ESLocal` or something along these lines?